snooty/intersphinx.py

```diff
@@ -127,8 +129,7 @@ def parse(cls, base_url: str, text: bytes) -> "Inventory":
             domain, role = domain_and_role.split(":", 1)
 
             # "If {dispname} is identical to {name}, it is stored as -"
-            if dispname == "-":
-                dispname = name
+            dispname = None if raw_dispname == "-" else raw_dispname
```

Comment:
Just wondering, where does the hyphen come from?

Reply:
I dug in quickly to ensure that I had this right, but sphinx uses a hyphen indicates that the label should be the same as the target name. For... some reason.